.github/workflows/terraform-ec2.yml

```diff
@@ -0,0 +1,187 @@
+name: Provision EC2 Instance with Terraform
+
+on: [push]
```

Comment:
Can we trigger the pipeline manually for now? I think the push will cause some problems since we don't have real blue-green deployment figured out yet.